src/main/resources/db/migration/V3.0__task_list_users.sql

```diff
@@ -0,0 +1,12 @@
+create table `task_list_users` (
```

Comment:
Forgive my novice grasp of Kotlin/Thymeleaf. Why do you need this additional table? (trying to understand, not provide design input)

Reply:
Ian has explained this to me in person. Thanks @iballard95